controls/anssi-minimal.yml

```diff
@@ -0,0 +1,404 @@
+policy: 'ANSSI-BP-028'
```

Comment:
The policy file should be renamed to `controls/anssi.yml`.

Reply:
But only after https://github.com/matejak/scap-security-guide/pull/5 is merged, :grin: